CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# ChugSplash Contributing Guide
+
+Hello, and thank you for your interest in contributing to ChugSplash!
+ChugSplash is a big project and we need all the help we can get.
+We've tried to make the contributing process as easy as possible.
+Please read through this guide for more information about setting up your development environment and making your first contribution.
+
+## Writing good issues
+
+ChugSplash tracks non-trivial work in GitHub issues.
+If you're planning to make a large PR, please make sure that there's a corresponding issue that tracks the work you're doing.
+With careful issue tracking, we can make sure that people aren't doing unnecessary work.
+Note that this isn't necessary for smaller PRs like typo fixes or minor bug fixes.
+
+When writing an issue meant to track a work item, please attempt to include as much detail as possible.
+Another contributor should be able to tackle the problem based only on the information included in the issue.
+Detailed issues are critical to keeping the ChugSplash project moving.
+
+## Setting up your development environment
+
+### Prerequisites
+
+You'll need to have the following pieces of software installed before you can start working on this repository:
+
+- [Git](https://git-scm.com/downloads)
+- [NodeJS](https://nodejs.org/en/download/)
+- [NVM](https://github.com/nvm-sh/nvm)
+- [Yarn 1.x](https://classic.yarnpkg.com/en/docs/install)
+
+### Cloning and installing
+
+Once you've installed the necessary prerequisites, you'll need to clone the ChugSplash monorepo and install its dependencies:
+
+```sh
+git clone https://github.com/smartcontracts/chugsplash.git
+cd chugsplash
+yarn install
+```
+
+### Install the correct version of Node using NVM
+
+Using `nvm`, install the correct version of NodeJS:
+
+```sh
+nvm use
+```
+
+### Et voilà
+
+You've set up everything necessary to start working on ChugSplash.
+Hopefully that wasn't too bad.
+
+## Common development stuff
+
+### Building
+
+```sh
+yarn clean
+yarn build
+```
+
+### Linting
+
+```sh
+yarn lint
+```
+
+### Testing
+
+```sh
+yarn test
+```
+
+## Conventional commits
```

Comment:
Perhaps it'd be helpful for new contributors to know the scopes that are commonly used in this project? For example, someone new might not know what "ct" means, and it isn't in the linked Conventional Commits page.

Reply:
Added